packages/cli/src/cmds/eval/eval-worker.ts

```diff
@@ -19,9 +18,18 @@ if (parentPort !== null) {
   parentPort.on('message', (message: { snapFilePath: string }) => {
     const { snapFilePath } = message;
 
-    new Compartment(getMockEndowments()).evaluate(
-      readFileSync(snapFilePath, 'utf8'),
-    );
+    const snapModule: any = { exports: {} };
```

Comment:
Why use `any` here?

Reply:
I didn't want to move the simple type from `BaseSnapExecutor.ts` to types package. I've strengthened it to `{exports?: any}` but since the unsafe snap can modify the exports property any way the want we need to do run-time checks and not allow compile-time checks to lull us into feeling of fake security. 

I've also releaxed the type in `BaseSnapExecutor.ts` and now they're consistent.